### PR TITLE
[#17611] move status-im.utils.universal-links.core

### LINKED ISSUE #17611: move status-im.utils.universal-links.core

### LINKED ISSUE #17794: move status-im.router.core
move status-im.router.core

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes: #17794 move status-im.router.core
fixes: #17611 move status-im.utils.universal-links.core

### CODE REVIEW

**Review by Parveshdhull — APPROVED** [review]

**Review by smohamedjavid — COMMENTED** [review]
The PR looks good!

Left one comment regarding translation resources.

File: src/status_im2/core.cljs [hunk 1]
```diff
@@ -1,9 +1,5 @@
 (ns status-im2.core
   (:require
-    ;; NOTE: Do NOT sort i18n-resources because it MUST be loaded first.
-    [status-im2.setup.i18n-resources :as i18n-resources]
-
-    #_{:clj-kondo/ignore [:unsorted-required-namespaces]}
```
Comment (smohamedjavid) [inline src/status_im2/core.cljs]: @flexsurfer - We should not remove this config. These changes need to be reverted. The translation resources need to be loaded first.

Right now, the app fails to load the translation.

Comment (flexsurfer) [reply src/status_im2/core.cljs]: yeah, thanks, I thought we excluded this file

**Review by smohamedjavid — APPROVED** [review]

**Comment by VolodLytvynenko** [comment]
Hi @flexsurfer Thank you for PR. take a look at a found issue.

## ISSUE 1: [Android] Universal links fail to open the app when accessed from external chat

### Steps:
1. Tap https://status.app/ link in any external chat  (Example: https://status.app/cc/Ow==#zQ3shYSHp7GoiXaauJMnDcjwU2yNjdzpXLosAWapPS4CFxc11)

### Actual result:
The app is not opened when the universal link is accessed from an external chat.


https://github.com/status-im/status-mobile/assets/52490791/32d91fab-fa32-480a-beeb-efe29cb6a065



### Expected result:
The app should open successfully when a universal link is accessed from an external chat.

### Logs:
[logs2.txt](https://github.com/status-im/status-mobile/files/13309240/logs2.txt)

**Comment by VolodLytvynenko** [comment]
> Hi @flexsurfer Thank you for PR. take a look at a found issue.
> 
> ## ISSUE 1: [Android] Universal links fail to open the app when accessed from external chat
> ### Steps:
> 1. Tap https://status.app/ link in any external chat  (Example: https://status.app/cc/Ow==#zQ3shYSHp7GoiXaauJMnDcjwU2yNjdzpXLosAWapPS4CFxc11)
> 
> ### Actual result:
> The app is not opened when the universal link is accessed from an external chat.
> 
>  external.mp4 
> ### Expected result:
> The app should open successfully when a universal link is accessed from an external chat.
> 
> ### Logs:
> [logs2.txt](https://github.com/status-im/status-mobile/files/13309240/logs2.txt)

@flexsurfer apologies. Ignore this issue. Not PR related

**Comment by VolodLytvynenko** [comment]
@flexsurfer thank you for PR. No issues from my side. Ready for merge

**Pull request merged**